packages/browser-sdk/src/providers/injected/index.ts

```diff
@@ -151,6 +155,31 @@ export class InjectedProvider implements Provider {
       throw new Error(`Invalid provider for injected connection: ${authOptions.provider}. Must be "injected"`);
     }
 
+    const requestedWalletId = authOptions.walletId || "phantom";
+    if (requestedWalletId === "phantom") {
+      // Default to Phantom injected provider
+      this.selectedWalletId = "phantom";
+      debug.log(DebugCategory.INJECTED_PROVIDER, "Selected Phantom injected wallet for connection", {
+        walletId: "phantom",
+      });
+    }
+
+    if (this.addressTypes.includes(AddressType.solana)) {
+      if (this.selectedWalletId !== "phantom") {
```

Comment:
Super nit: we don't need separate conditions, e.g.
```suggestion
    if (this.addressTypes.includes(AddressType.solana) && this.selectedWalletId !== "phantom") {
```